src/collectors/cephstats/cephstats.py

```diff
@@ -8,11 +8,40 @@
 import re
 import os
 import sys
-
 sys.path.insert(0, os.path.join(os.path.dirname(os.path.dirname(__file__)),
                                 'ceph'))
 from ceph import CephCollector
 
+patternchk = re.compile(r'\bclient io .*')
+numberchk = re.compile(r'\d+')
+
+
+# This is external to the CephCollector so it can be tested
+# separately.
+def process_ceph_status(output):
+    if 'client io' not in output:
+        return {}
+    res = patternchk.search(output)
+    if not res:
+        return {}
+    ceph_stats = res.group()
+    if not ceph_stats:
+        return {}
+    rd = wr = iops = None
+    rd = numberchk.search(ceph_stats)
+    if rd:
```

Comment:
I'd check these against an explicit `if rd is None:`, `if wr is not None:` and `if iops is not None:`.